### PR TITLE
make colossal improvements to user experience

### PR DESCRIPTION
**Pull request by 0us**
- update font to be more hacker friendly
- accessability improvements
- general style and typography improvements
- center items in the columns
- add hover effect on list items

Hope you enjoy these changes, and remember to always live your best life!
Best Regards
Team Natron

### hover effect 
bilde

### before/after

bilde
bilde

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -40,6 +45,8 @@ h2,
 h3,
 h4 {
   text-align: center;
+  color: #dee3e3;
```
Comment (Mordi490) [inline css/main.css:48]: Too bright, use colors from the Solarized theme

Comment (0us) [reply css/main.css:48]: i refuse

File: css/main.css [hunk 2]
```diff
@@ -20,7 +20,12 @@ palette from generator 2
     url("/static/fonts/ttf/Overpass-LightItalic.ttf");
 }
 
+a > img {
+  margin-left: 1rem;
+}
+
 body {
+  font-family: "Roboto Mono", sans-serif;
```
Comment (Mordi490) [inline css/main.css:28]: suggestion: Inconsolata

File: css/main.css (hunk 1 above)
Comment (0us) [reply css/main.css]: Darkened the titles. This is an excellent compromise. The user experience is now even better than before. Honestly i feel pretty great about this

File: css/main.css [hunk 3]
```diff
@@ -119,17 +148,7 @@ img {
...
-
-  li {
-    list-style-type: none;
-    padding: 1em;
-    display: flex;
-    flex-flow: column wrap;
-    justify-content: flex-start;
-    align-items: flex-start;
-    max-width: 15em;
+    color: #dee3e3;
```
Comment (Mordi490) [inline css/main.css:151]: muh color scheme

File: css/main.css (hunk 2 above)
Comment (0us) [reply css/main.css:28]: looks a bit thin. but up to you
bilde

File: css/main.css (hunk 3 above)
Comment (0us) [reply css/main.css:151]: the color stays

File: css/main.css (hunk 1 above)
Comment (0us) [reply css/main.css]: You can see how this looks here https://github.com/Mordi490/mordi490.github.io/pull/7#discussion_r641864601

**Review by Mordi490 — CHANGES_REQUESTED** [review]
Go with Inconsolata
look at this https://github.com/Mordi490/mordi490.github.io/issues/1#issuecomment-850810881 and we're good

**Comment by 0us** [comment]
> 
> 
> Go with Inconsolata
> look at this [#1 (comment)](https://github.com/Mordi490/mordi490.github.io/issues/1#issuecomment-850810881) and we're good

have to close this and open a new pr because my fork disappeared

**Pull request closed without merging**